MetaMask/metamask-mobile
# feat: Update onboarding copy

## **Description**

Updates the onboarding text.

This PR is against the new-privacy-policy-toast since they need to go together

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/2220

## **Manual testing steps**

1. Start onboarding a new user
2. Check that the notice has the new copy

## **Screenshots/Recordings**

<img width="385" alt="Screenshot 2024-04-15 at 14 25 19" src="https://github.com/MetaMask/metamask-mobile/assets/11148144/dc270589-1b93-4627-bac1-87563afd1086">


### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
